reference/UiTPAS.v2.json

```diff
@@ -333,153 +353,1183 @@
+          },
+          "member": {
+            "type": "array",
+            "items": {
+              "$ref": "#/components/schemas/Passholder"
+            }
+          }
+        }
+      },
+      "Passholder": {
+        "title": "Passholder",
+        "type": "object",
+        "x-tags": [
+          "Models"
+        ],
+        "properties": {
+          "id": {
+            "type": "string"
+          },
+          "name": {
+            "type": "string"
+          },
+          "firstName": {
+            "type": "string"
+          },
+          "inszNumber": {
+            "type": "string"
+          },
+          "cardSystemMemberships": {
+            "type": "array",
+            "items": {
+              "$ref": "#/components/schemas/CardSystemMembership"
+            }
+          },
+          "email": {
+            "type": "string"
+          },
+          "creationDate": {
+            "type": "string",
+            "format": "date-time"
+          }
+        }
+      },
+      "CardSystemMembership": {
+        "title": "CardSystemMembership",
+        "type": "object",
+        "x-tags": [
+          "Models"
+        ],
+        "properties": {
+          "cardSystem": {
+            "$ref": "#/components/schemas/CardSystem"
+          },
+          "uitpasNumber": {
+            "type": "string"
+          },
+          "status": {
+            "type": "string",
+            "enum": [
+              "ACTIVE",
+              "BLOCKED"
+            ]
+          },
+          "socialTariff": {
+            "type": "object",
+            "properties": {
+              "endDate": {
+                "type": "string",
+                "format": "date-time"
+              },
+              "inGracePeriod": {
+                "type": "boolean"
+              },
+              "expired": {
+                "type": "boolean"
+              }
+            }
+          }
+        }
+      },
+      "CardSystem": {
+        "title": "CardSystem",
+        "type": "object",
+        "x-tags": [
+          "Models"
+        ],
+        "properties": {
+          "id": {
+            "type": "integer"
+          },
+          "name": {
+            "type": "string"
+          }
+        },
+        "required": [
+          "id"
+        ]
+      },
+      "PassholderUpdateRequest": {
+        "title": "PassholderUpdateRequest",
+        "type": "object",
+        "description": "Request object to update a passholder",
+        "properties": {
+          "schoolId": {
+            "type": "string"
+          }
+        },
+        "x-tags": [
+          "Models"
+        ]
+      },
+      "ReportStatus": {
+        "type": "object",
+        "title": "ReportStatus",
+        "description": "The status of a started report export.",
+        "x-examples": {},
+        "x-tags": [
+          "Models"
+        ],
+        "properties": {
+          "status": {
+            "type": "string",
+            "enum": [
+              "STARTED",
```

Comment:
@bertramakers opnieuw een kwestie: wat als de export faalde? is dat een type enum 'FAILED' of moet de /status request dan een 400 geven en zit de echte reden (die aan de gebruiker getoond kan worden) in de error.detail?

Reply:
`FAILED` status + extra veldje met uitleg is beter denk ik, anders kan het lijken alsof er iets mis was met de HTTP request om de status op te vragen (400 = bad request, maar de request in kwestie is niet slecht, de export die eerder is gestart is gewoon gefaald)